### PR TITLE
Bandeau beta

### PR DESCRIPTION
**Pull request by abulte**
Fix https://github.com/etalab/data.gouv.fr/issues/296

TODO: iterate on wording, fink a link for "apprenez-en plus" @agarrone 

Nice to have: click to dismiss (but it goes away when scrolling down on mobile).

Capture d’écran 2021-07-01 à 12 37 10
Capture d’écran 2021-07-01 à 12 36 59

### CODE REVIEW

File: udata_gouvfr/theme/gouvfr/templates/header.html [hunk 1]
```diff
@@ -1,11 +1,20 @@
+<div class="bg-blue-350">
+    <div class="hagrid banner">
+        <div class="row align-items-center text-align-center-sm">
+            <p class="text-white">
+                Bienvenue sur la nouvelle version <i>beta</i> de data.gouv.fr&nbsp;! <a href="">Apprenez-en plus</a> ou revenez <a href="https://legacy.data.gouv.fr">à l'ancienne version.</a>
```
Comment (maudetes) [inline udata_gouvfr/theme/gouvfr/templates/header.html:5]: We should write it in English and add the French translation I guess.

Comment (abulte) [reply udata_gouvfr/theme/gouvfr/templates/header.html:5]: Not sure this is worth it for such a temporary thing.

Comment (maudetes) [reply udata_gouvfr/theme/gouvfr/templates/header.html:5]: Depends if we communicate on the beta at an international level ?

Comment (abulte) [reply udata_gouvfr/theme/gouvfr/templates/header.html:5]: True. I'll change it.

**Review by maudetes — APPROVED** [review]

**Pull request merged**